### ISSUE TITLE
Use the splatmap and the rock shader in the main scene

### ISSUE THREAD

**Issue by yaelatletl** [moonwards1/Moonwards-Virtual-Moon#65]
As the title says, implement what's already done with it

**Comment by Moonwards** [comment]
I think it needs a little tweaking so the different textures blend realistically when seen from a distance. 

Otherwise, this is a big improvement and I'm sure it would be satisfying to finally have it implemented.

**Comment by Moonwards** [comment]
This is waiting on completion of the new crater height map by Duriel which will be set up for LOD. 

Then we may well want to reconsider our approach to this, as there are new factors like the shader code posted by Gyrth today.

[https://www.reddit.com/r/Unity3D/comments/dhr5g2/i_made_a_stochastic_texture_sampling_shader/](url)
